.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -14,7 +14,7 @@ body:
     id: reproduce
     attributes:
       label: Steps to reproduce the bug
-      description: Describe how to reproduce this issue.
+      description: Describe how to reproduce this issue. If the issue is found only in your project, provide a link to a public repo or code preview that demonstrates this issue.
```

Comment:
**Question:** What do you think about making the demo its own, non-required, field? I think it'd bring additional attention to the ask and could see a benefit of separating it from the testing instructions so users could provide the demo, and then specifically outline the steps to reproduce.

Reply:
My instinct is to leave it as is in a single field to see if it helps before adding additional fields. That said, happy to defer to the group if we decide we want an additional field. @mejiaj, do you have a preference?